### DISCUSSION TITLE
Request feature for exclude filters and only tags for sync.

### DISCUSSION THREAD

**Discussion by Dnarts** [thesamim/TickTickSync#69] [Ideas]
Hello! First of all. Thank you VERY much. 
1. **Tags for synchronization**
2. **No projects -> indicates that only tags are synchronized** i really need it for backwards process. (like I need to think about something and expand my vision for the tasks tagged as such, not every single task on TickTick)
3. Exclude folders
4. Exclude tags
5. Include an option to prevent synchronization of completed tasks that were created outside of Obsidian (old ones, as this trigger unnecessary sync cycles from the start, i dont know why, everytime when im running obsidian)

Adding this feature would give users more control.
Please consider adding this useful feature in a future update. Thank you again for this great extension!

**Comment by Dnarts** [comment]
Also, there is some bug that appends few finished tasks ([x]) to text which I edit when addon is getting synchronized

**Comment by thesamim** [comment]
The functionality kind of exists :
![image](https://github.com/thesamim/TickTickSync/assets/5315625/223b6dab-1dd9-4536-8bd1-6118b86ae46e)

So conceivably, you could put tasks you want to sync in a certain folder, or tag them with a certain tag. Or both.

>     2. **No projects -> indicates that only tags are synchronized** i really need it for backwards process. (like I need to think about something and expand my vision for the tasks tagged as such, not every single task on TickTick)
> 
Unless you have "Full Vault Sync" turned on, only tasks marked with `#ticktick` are synced.  So, I'm not really understanding the ask.

>     3. Exclude folders
>     4. Exclude tags
> 
We have "include*" right now...

>     5. Include an option to prevent synchronization of completed tasks that were created outside of Obsidian (old ones, as this trigger unnecessary sync cycles from the start, i dont know why, everytime when im running obsidian)
> 
If they don't have the `#ticktick` tag on them, they should not be touched. Unless you have "Full Vault Synchronization" turned on of course. 

> Also, there is some bug that appends few finished tasks ([x]) to text which I edit when addon is getting synchronized 
>
Sorry, don't understand. Could you give me an example please?

> Please consider adding this useful feature in a future update. Thank you again for this great extension!
>
Will consider adding more specific controls for a future release.

You're welcome!
